torrents/management/commands/add_snatched_by_tracker.py

```diff
@@ -0,0 +1,188 @@
+from bencode import bdecode
+import os
+import shutil
+
+from django.core.management import BaseCommand
+
+from torrents.download_locations import format_download_path_pattern
+from torrents.models import TorrentInfo, TorrentFile, Realm, Torrent
+from torrents.add_torrent import add_torrent_from_tracker, fetch_torrent
+from torrents.exceptions import RealmNotFoundException
+
+from trackers.registry import TrackerRegistry
+
+class ImportedTorrentMover():
+    # store_files_hook for add_missing_from_tracker
+
+    # base_dir:   Location where it looks /mnt/media/music
+    # folder:     Name of the folder
+    # sub_dir:    All torrent files relative to the folder
+    # to_copy     True - Copy the files in, False - Move them in
+    def __init__(self, base_dir=None, sub_dir=None, folder=None, to_copy=True):
+        self.base_dir = base_dir
+        self.sub_dir = sub_dir
+        self.folder = folder
+        self.to_copy = to_copy
+    
+    # Moves all torrent contents to new download_path
+    def move_files(self, torrent_info, download_path):
+        source_base_dir = self.base_dir
+        dest_base_dir = download_path
+
+        if self.folder:
+            source_base_dir = os.path.join(source_base_dir,self.folder)
+            dest_base_dir = os.path.join(dest_base_dir, self.folder)
+        for path in self.sub_dir:
+            source_path = os.path.join(source_base_dir, path)
+            dest_path = os.path.join(dest_base_dir, path)
+            if not os.path.exists(os.path.dirname(dest_path)):
+                os.makedirs(os.path.dirname(dest_path))
+            if self.to_copy:
+                shutil.copy(source_path, dest_path)
+            else:
+                os.rename(source_path, dest_path)
+        return
+
+
+class Command(BaseCommand):
```

Comment:
I think code comments are good, but what's better is documentation for the user - what the command does, how's it used, what it expects, etc. You can make use of the "help" attribute in Django:

```
class Command(BaseCommand):
    help = 'some short or long description'
```